fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/Auth.java

```diff
@@ -1407,6 +1408,58 @@ private void getUserAuthInfo(List<List<String>> userAuthInfos, UserIdentity user
         userAuthInfos.add(userAuthInfo);
     }
 
+    public static boolean isAdminPriv(GlobalPrivTable userGlobalPrivTable) {
+        PrivBitSet globalPrivs = new PrivBitSet();
+        for (PrivEntry pe : userGlobalPrivTable.getEntries()) {
+            globalPrivs.or(pe.privSet);
+        }
+        if (globalPrivs.satisfy(PrivPredicate.ADMIN)) {
+            return true;
+        }
+        return false;
+    }
+
+    public void getUserRoleWorkloadGroupPrivs(List<List<String>> result, UserIdentity currentUserIdentity) {
+        readLock();
+        try {
+            boolean isCurrentUserAdmin = isAdminPriv(getUserGlobalPrivTable(currentUserIdentity));
```

Comment:
Can `checkGlobalPriv (current UserIdentity, PrivPredicte. ADMIN)` be used instead?